documentation/docs/install-pmm/install-pmm-client/connect-database/mongodb.md

```diff
@@ -1,50 +1,56 @@
 # Connect MongoDB instance
 
-How to set up PMM to monitor a [MongoDB] or [Percona Server for MongoDB] database instance.
+Connect a MongoDB instance to PMM to monitor a [MongoDB] or [Percona Server for MongoDB] database server.
 
-## Before you start
+## Prerequisites
 
-Check that:
+Before you start, ensure you have:
 
-- [PMM Server is installed](../../install-pmm-server/index.md) and running with a known IP address or hostname accessible from the client node.
-- [PMM Client is installed](../../install-pmm-client/index.md) and the [nodes are registered with PMM Server](../../register-client-node/index.md).
-- You have superuser (root) access on the client host.
-- You have `adminUserAnyDatabase` or superuser role privilege to any database servers that you want to monitor.
-- Your MongoDB server is version 4.0 or higher.
+- [PMM Server installed](../../install-pmm-server/index.md) and running with a known IP address or hostname accessible from the Client node.
+- [PMM Client installed](../../install-pmm-client/index.md) and the [nodes are registered with PMM Server](../../register-client-node/index.md).
+- Superuser (root) access on the Client host.
+- `adminUserAnyDatabase` or superuser role privilege on the MongoDB database servers that you want to monitor.
+- MongoDB server version 4.0 or higher.
 
 ## Create PMM account and set permissions
 
-We recommend using a dedicated account to connect PMM Client to the monitored database instance.
+We recommend using a dedicated account to connect PMM Client to the monitored database instance. The permissions required depend on which PMM features you plan to use.
 
-Run the example codes below in a `mongo` session to:
+Run the example commands below in a mongo shell session to:
 
--  create custom roles with the privileges required for creating/restoring backups and working with Query Analytics (QAN)
--  create/update a database user with these roles above, plus the built-in  `clusterMonitor` role
+-  Create custom roles with the privileges required for creating/restoring backups and working with Query Analytics (QAN).
+-  Create/update a database user with these roles, plus the built-in  `clusterMonitor` role.
   
-!!! caution alert alert-warning ""
+!!! caution alert alert-warning "Important"
     Values for username (`user`) and password (`pwd`) are examples. Replace them before using these code snippets.
 
-=== "Create a role with monitoring and QAN privileges"
-    This role grants the essential minimum privileges needed for monitoring and QAN. 
+=== "Monitoring and QAN privileges"
+    This role grants the essential minimum privileges needed for monitoring and QAN:
+
         ```{.javascript data-prompt=">"}
         db.getSiblingDB("admin").createRole({
         "role": "explainRole",
```

Comment:
pmmMonitor?

Reply:
what do yo mean here?

Reply:
should we rename role from explainRole to pmmMonitor as it includes permissions required not only for explain, but for mongodb_exporter indexstats and dbstats as well?

Reply:
I think it should be safe to rename it in docs for new installations.

Reply:
@catalinaadam ^